src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -31,7 +31,7 @@ let nbCount: number = 0;
             <clr-dg-string-filter
                     *ngIf="field && !customFilter"
                     [clrDgStringFilter]="registered"
-                    [(clrFilterValue)]="filterValue"></clr-dg-string-filter>
+                    (clrFilterValueChange)="filterValueChange.emit($event)"></clr-dg-string-filter>
```

Comment:
Why did we lose the `@Input` `clrFilterValue` here? With this change, we are only binding on the `Output`, no? Does pre-setting a filter value work after this change?

Reply:
We can still pre-set the filter value. I noticed that having two-way binding with `[(clrFilterValue)]` on `clr-dg-string-filter` had no impact. Inside the clr-`dg-column`, we actually set the filter value programmatically [here](https://github.com/Shijir/clarity/blob/bf77f817c6b86cb6ebc6740d1abbf134ab991653/src/clr-angular/data/datagrid/datagrid-column.ts#L318). 

To see whether it had impact or not, we actually tested the pre-setting filter value before. And I haven't made changes to those pre-existing tests and they are still passing with the changes I made. Also, you can see the filter was pre-set with a value, "d" in the [demo](http://2004-after-fix.surge.sh/datagrids) (in the "Sort and Filter" section).

So we didn't need to set the value to `clr-dg-string-filter` using template bindings.